staging/kube-oidc-proxy/templates/deployment.yaml

```diff
@@ -51,6 +51,8 @@ spec:
           - "--oidc-username-claim=$(OIDC_USERNAME_CLAIM)"
           {{- if or .Values.oidc.caPEM .Values.oidc.caSecretName }}
           - "--oidc-ca-file=/etc/oidc/oidc-ca.pem"
+          {{- else if and .Values.oidc.caCertPemHostPath (not .Values.oidc.caPEM) (not .Values.oidc.caSecretName) }}
```

Comment:
The `not` sections are not needed as this is an `else if`

Reply:
Also, I think we should also have a way to *not* specify this flag if the user simply want to use the ca bundle in the image. i.e., even if `caCertPemHostPath` is not set, we still need a way allowing users to not setting this flag.

Thus, we still need `caKubernetesDefault` flag i suggested below.